_pages/attend/accomodation.md

```diff
@@ -1,37 +1,12 @@
 ---
 layout: page
-title: Accomodation
+title: Accommodation
 snake: rainbow
-permalink: /attend/accomodation
-sponsors: true
+permalink: /attend/accomodation/
+card: schedule.png
```

Comment:
Are you sure you want `schedule.png` here? That file doesn't exist anywhere in the repo.

Reply:
Very good point.